uber/config.py

```diff
@@ -273,7 +271,6 @@ def CURRENT_ADMIN(self):
     def HTTP_METHOD(self):
         return cherrypy.request.method
 
-    @property
     def SUPPORTER_COUNT(self):
```

Comment:
+1 a bit omhbity

On Fri, Aug 12, 2016, 00:35 Dominic Cerquetti <notifications@github.com>
wrote:

> In uber/config.py
> <https://github.com/magfest/ubersystem/pull/1963#discussion_r74546782>:
>
> > @@ -273,7 +271,6 @@ def CURRENT_ADMIN(self):
> >      def HTTP_METHOD(self):
> >          return cherrypy.request.method
> >
> > -    @property
> >      def SUPPORTER_COUNT(self):
>
> i feel like even though we can't use the decorator form here, we should
> probably mark these with a comment like:
>
> # IMPORTANT - Cherrypy Request-specific caching in place here, see request_cached_property()
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/magfest/ubersystem/pull/1963/files/52fb90dc5f5d0e4946702c6dec01f4fd8aef3924#r74546782>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AG3WB1Yed7RbbE-o_iM2wIPXfpDHqtZPks5qfBQZgaJpZM4JiwZw>
> .
>